doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -60,13 +60,13 @@ $ astartectl housekeeping realms ls --housekeeping-url http://localhost:4001/ -k
 
 ## Install an interface
 
-We will use [Astarte's Qt5 Stream Generator](https://github.com/astarte-platform/stream-qt5-test) to feed data into Astarte. Clone the repository, as we will have to install its `org.astarteplatform.Values` interface into our new realm. To do that, we can use `astartectl` again:
+We will use [Astarte's Qt5 Stream Generator](https://github.com/astarte-platform/stream-qt5-test) to feed data into Astarte. Clone the repository, as we will have to install its `org.astarte-platform.genericsensors.Values` interface into our new realm. To do that, we can use `astartectl` again:
```

Comment:
No need to clone the repo anymore since standard-interfaces are already in astarte repo